frontend/src/features/projects/spl/tabs/CloseOutFormTab.tsx

```diff
@@ -37,7 +36,7 @@ const CloseOutFormTab: React.FunctionComponent<ICloseOutFormTabProps> = ({
       <Form.Row>
         <Col>
           <ProjectNotes
-            field={`notes[${NoteTypes.SalesHistory}].note`}
```

Comment:
Using array positioning is an invalid implementation and will result in data corruption